pkgs/development/libraries/libogg/default.nix

```diff
@@ -1,23 +1,15 @@
 { lib, stdenv, fetchurl, fetchpatch }:
 
 stdenv.mkDerivation rec {
-  name = "libogg-1.3.4";
+  name = "libogg-1.3.5";
 
   src = fetchurl {
     url = "http://downloads.xiph.org/releases/ogg/${name}.tar.xz";
-    sha256 = "1zlk33vxvxr0l9lhkbhkdwvylw96d2n0fnd3d8dl031hph9bqqy1";
+    sha256 = "01b7050bghdvbxvw0gzv588fn4a27zh42ljpwzm4vrf8dziipnf4";
   };
 
   outputs = [ "out" "dev" "doc" ];
 
-  patches = lib.optionals stdenv.isDarwin [
-    # Fix unsigned typedefs on darwin. Remove with the next release https://github.com/xiph/ogg/pull/64
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Good catch! Done.